automation/terraform/modules/kubernetes/buildkite-agent/helm.tf

```diff
@@ -58,7 +58,7 @@ locals {
     },
     {
       "name" = "KUBE_CONFIG_PATH"
-      "value" = "~/.kube/config"
+      "value" = "/root/.kube/config"
```

Comment:
default value since agent container in k8s runs as root by default tho only has access to its own fs